index.html

```diff
@@ -1,569 +1,476 @@
-              fill="currentColor"
-            />
-          </svg>
-        </a>
-      </div>
-
-      <div class="tools-summery">
-        <div class="header">
-          <h2>languages</h2>
-          <i class="bx bx-chevron-down"></i>
+        <div class="about-infos">
+            <div class="info-description">
+                <h2>About Myself</h2>
+                <p class="">Hello I’m a software developer! I can help you build a product , feature or website. Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hesitate to contact me.
+                </p>
+            </div>
+
+
+            <div class="contact">
+                <a href="">Get my resume</a>
+            </div>
+
+            <div class="showcase-contact-icons">
+                <a href="">
+                    <svg class="showcase-icon" width="20" height="20" viewBox="0 0 20 20" fill="currentColor"
+                        xmlns="http://www.w3.org/2000/svg">
+                        <path
```

Comment:
- [ ] **Required:** As mentioned by my colleague, your 'About Myself' section is not arranged as shown on the figma file, as your are missing the 'LET's CONNECT' anchor tag and your `showcase-contact-icons` should be above the 'Get my resume' button